.github/workflows/ci_push.yml

```diff
@@ -112,6 +112,8 @@ jobs:
       run: |
         .github/workflows/install_deps.sh ipopt
         pip install -e .[test,ipopt,dlib,pyswarm,cmaes,nlopt,fides]
+        sudo apt install libopenmpi-dev
```

Comment:
Dunno if @yannikschaelte has an opinion on including that in `install_deps.sh` 2 lines above? Either fine for me.

Reply:
May be nicer to include in install_deps.sh, as I also use that to install things locally

Reply:
added to install_deps.sh